src/main/java/com/hyperwallet/clientsdk/model/HyperwalletDocumentRejectReason.java

```diff
@@ -0,0 +1,29 @@
+package com.hyperwallet.clientsdk.model;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlRootElement;
+
+@XmlRootElement
+@XmlAccessorType(XmlAccessType.FIELD)
+public class HyperwalletDocumentRejectReason {
```

Comment:
do we need this here?
@JsonFilter(HyperwalletJsonConfiguration.INCLUSION_FILTER)